### DISCUSSION TITLE
README, CONTRIBUTING doc request

### DISCUSSION THREAD

**Discussion by njayinthehouse** [jsoftware/jsource#236] [General]
Seems like an interesting project, if you could provide some resources to get started, people might like to submit some ideas. :)

**Comment by danuker** [comment]
Here's the Wiki linked from the [main site](https://www.jsoftware.com/):
https://code.jsoftware.com/wiki/Guides/Getting_Started

And here's [Rosetta Code](http://rosettacode.org/) exercises in J, where you can see examples of how it's used:
https://github.com/acmeism/RosettaCodeData/tree/master/Lang/J

I found J by analyzing programming language density (there are many short examples; but I later found out many of them are only parts of the program).
